src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { snazzyButtonStyle } from './helpers';
```

Comment:
You've imported the style, good, don't delete this line. 
You need to use it below.